modular_skyrat/modules/loadouts/loadout_items/loadout_datum_masks.dm

```diff
@@ -97,6 +97,9 @@ GLOBAL_LIST_INIT(loadout_masks, generate_loadout_items(/datum/loadout_item/mask)
 /*
 *	MISC
 */
+/datum/loadout_item/mask/surgical //I'm putting this closer to where it used to be because people WILL just assume its gone otherwise
```

Comment:
```suggestion
/datum/loadout_item/mask/surgical
```
(Now that reviewers have read the comment it's not needed and will stick out in-code later on; someone could think "Closer to where it used to be? Where did it used to be? What's wrong here?")
Not important but hey it keeps a PR from goin' stale 👉 👉